misc/config_tools/schema/types.xsd

```diff
@@ -147,6 +147,24 @@ higher value (lower severity) are discarded.</xs:documentation>
   </xs:restriction>
 </xs:simpleType>
 
+<xs:simpleType name="policyType">
+  <xs:annotation>
+    <xs:documentation>A string specifying the CPU frequency policy type:
+
+- ``Performance``: CPU runs at its maximum frequency. Enable hardware autonomous frequency selection if HWP is presented.
```

Comment:
Change "if HWP is presented" to "if the system supports [full term] (HWP)". Replace [full term] with the term that HWP stands for (is it Hardware P-state?).

Reply:
The full term is ‘Hardware-Controlled Performance States’. I have changed it and force pushed.